### DISCUSSION TITLE
Do you plan on making a guide for users who havent done much or any coding before?

### DISCUSSION THREAD

**Discussion by AlienGuardians** [davidhozic/discord-advertisement-framework#81] [Q&A]
Do you plan on making a guide for users who havent done much or any coding before. I myself am so lost. I downloaded the latest version of python and the command (python -m pip install discord-advert-framework) is not working, i just keep getting syntax error. I get syntax error with everything i try and do in python, im aware that this is a python problem and not your problem but i would lovesome help because its becoming quite frustrating. I would love a guide! I would even pay for it..

**Comment by davidhozic** [comment]
Which os do you have, Windows, Linux or Mac?
Maybe just try `pip install discord-advert-framework`.

**Reply by davidhozic** [reply]
Did you solve it? @AlienGuardians

**Comment by tantypanty** [comment]
Hey @davidhozic I am struggling what to do now, I ran this python -m pip install discord-advert-framework and everything installed normally but I am lost what to do now

**Reply by davidhozic** [reply]
I'll probably make some videos after I'm finished with the SQL logging module..

In the mean time you can get started with examples, should be pretty straight forward:
https://github.com/davidhozic/discord-advertisement-framework/tree/master/Examples

Basically you create a list of [framework.GUILD](https://github.com/davidhozic/discord-advertisement-framework/blob/master/DOC.md#frameworkguild) objects
and then create [framework.TextMESSAGE](https://github.com/davidhozic/discord-advertisement-framework/blob/master/DOC.md#frameworktextmessage) objects if you want to send text.
After that you need to give the [framework.run](https://github.com/davidhozic/discord-advertisement-framework/blob/master/DOC.md#frameworkrun) function the token where if the token is from an user account and not a bot, you also need to set the is_user option to True.

Let me know if you still need help after trying with examples

**Comment by davidhozic** [comment]
> Do you plan on making a guide for users who havent done much or any coding before. I myself am so lost. I downloaded the latest version of python and the command (python -m pip install discord-advert-framework) is not working, i just keep getting syntax error. I get syntax error with everything i try and do in python, im aware that this is a python problem and not your problem but i would lovesome help because its becoming quite frustrating. I would love a guide! I would even pay for it..

A guide now exists, and also a GUI (with static definition of accounts, guilds, ...)
